src/Sites/Site.php

```diff
@@ -158,6 +158,26 @@ public function appStatus()
     }
 
     /**
+     * Site working directory.
+     *
+     * @return string | null
+     */
+    public function directory()
+    {
+        return $this->getData('directory');
+    }
+
+    /**
+     * Site creation status.
+     *
+     * @return string | null
+     */
+    public function siteStatus()
```

Comment:
We already have `ApiResource::status` method which retrieves `status` field value from the payload. Since `Site` class extends from `ApiResource`, said method is already available here. So, I guess this can be removed, too.